static/bidder-info/orbidder.yaml

```diff
@@ -6,3 +6,11 @@ capabilities:
   app:
     mediaTypes:
       - banner
+  site:
+    mediaTypes:
+      - banner
+      - native
```

Comment:
Requesting to please add json test case for above media types.

Reply:
I will add the test cases asap

Reply:
@gargcreation1992, thanks for the hint with the test.
Do not see this coming, that we had a todo at aout bid-adapter.

Fix setting BidType and adding tests with PR: https://github.com/prebid/prebid-server/pull/2924/commits/8902398c03a8089eee7e84471518c2df17bc433c